setup.py

```diff
@@ -45,6 +45,7 @@
     install_requires=(
         "google-api-core[grpc] >= 1.22.2, < 2.0.0dev",
         "proto-plus >= 1.10.1",
+        "pandas>=1.0.0",
```

Comment:
This should be added to a extras_require field adjacent to install_requires ie:

```python
install_requires=(...),
extras_require={
    "full": ["pandas>=1.0.0"]
}
```

Then the install with pandas would be 

```
pip install google-cloud-aiplatform[full]
```

Reply:
done